herd/RISCVSem.ml

```diff
@@ -255,7 +255,8 @@ module
       let build_semantics _ ii =
         M.addT (A.next_po_index ii.A.program_order_index)
           begin match ii.A.inst with
-          | RISCV.INop -> B.next1T ()
+          | RISCV.INop-> B.next1T ()
+          | RISCV.Ret -> M.unitT () >>! B.Exit
```

Comment:
As far as I could check, RISCV RET is jump tp the content of `ra` (similar to AArch64 ret).

Reply:
Guarding under the `Telechat` variant